language/json-language-service/src/__tests__/__snapshots__/service.test.ts.snap

```diff
@@ -168,7 +168,7 @@ exports[`player language service > validation > throws AssetWrapper errors 1`] =
     "severity": 2,
   },
   {
-    "message": "View Validation Error - value: Does not match any of the expected types for type: 'AssetWrapperOrSwitch'",
+    "message": "View Validation Error - value: Received unexpected types for type: 'AssetWrapperOrSwitch'",
```

Comment:
Maybe a bit nitpicky but `Received unexpected` seems a bit more vague than the original message. IMO it sounds like the validator received something unexpected rather than what it did receive was wrong